### PR TITLE
Skin fixes and additions

### PR DESCRIPTION
**Pull request by andromeda-Z**

### CODE REVIEW

**Comment by aokod** [comment]
I fail to understand how some of these changes improve the stylesheet.  Here's an example:

```css
font-family:monospaced,Courier New,sans,serif,arial,helvetica,sans-serif;
```

Making the base font monospaced wouldn't help with readability I don't think.  If some of these commits are a mistake on your behalf or I've misunderstood them don't hesitate to clear things up.  :)

**Comment by andromeda-Z** [comment]
> I fail to understand how some of these changes improve the stylesheet. Here's an example:
> 
> ```css
> font-family:monospaced,Courier New,sans,serif,arial,helvetica,sans-serif;
> ```
> 
> Making the base font monospaced wouldn't help with readability I don't think. If some of these commits are a mistake on your behalf or I've misunderstood them don't hesitate to clear things up. :)

Yes, that was accidental, it was meant to go at the end of the list in case the user has no sans-serif fonts. I'll fix that right now

File: skins/Plastic/styles.css [hunk 1]
```diff
@@ -40,7 +40,7 @@ a#nav-exit {border-top:0; border-bottom:0; border-left:1px solid; border-right:0
 /* Tags & gambits */
 #tagArea {border:1px dashed #bbb}
 #tags a {color:#777}
-#gambits a {color:#00f}
+#gambits a {color:#fff}
```
Comment (aokod) [inline skins/Plastic/styles.css:43]: May I ask why changing links from blue to solid white helps with readability?

File: skins/Plastic/styles.css [hunk 2]
```diff
@@ -64,7 +64,7 @@ a#nav-exit {border-top:0; border-bottom:0; border-left:1px solid; border-right:0
 a.star0:hover {color:#fe7}
 .star1, a.star1:hover {color:#fc0}
 a.star0:active, a.star1:active {color:#e4b600}
-.starred td {background:#ffa}
+.starred td {background:#ffb}
```
Comment (aokod) [inline skins/Plastic/styles.css:67]: ?

File: skins/base.css [hunk 3]
```diff
@@ -1,13 +1,13 @@
 @media all {
 /* Main tags */
-body {font-family:arial,helvetica,sans-serif; font-size:75%; margin:0; -webkit-text-size-adjust:none; padding-top:0px!important; min-width:320px}
+body {font-family:Courier New,sans,serif,arial,helvetica,sans-serif,monospaced; font-size:75%; margin:0; -webkit-text-size-adjust:none; padding-top:0px!important; min-width:320px}
```
Comment (aokod) [inline skins/base.css:3]: This is in the base stylesheet, so I feel that forcing skins to work around a specific typeface might not be the greatest idea in the way of letting skins be customizable.

**Pull request merged**